src/marqo/index.py

```diff
@@ -122,8 +122,10 @@ def search(self, q: str, searchable_attributes: Optional[List[str]] = None,
         """Search the index.
 
         Args:
-            q: string to search, or a pointer/url to an image if the index has
-                treat_urls_and_pointers_as_images set to True
+            q: String to search, or a dictionary of weighted strings to search
```

Comment:
is there a range for the weights? like (-5, +5) something.

Reply:
nope